totes

```diff
@@ -36,7 +50,10 @@ get_note_to_render() {
 }
 
 render_command="$(get_render_command)"
-note_to_render="$(get_note_to_render "$@")"
+# propagate errors
+note_to_render="$(get_note_to_render)" || exit $?
+printf "note: %s\n" "$note_to_render"
```

Comment:
did you mean to leave these two lines in?

Reply:
whoops, I do mean to the leave the `|| exit $?` as that correctly propagates errors from `get_note_to_render` but did not mean to leave in the `printf`, will remove

Reply:
oh I see by two lines you meant the two `printf`s. In the future you can select multiple lines by clicking and dragging:

![multilline-select](https://user-images.githubusercontent.com/9750687/71771252-fec98280-2eec-11ea-8f3d-d4440c6a6545.gif)

Reply:
<thumbs up>